src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const object of actions) {
+    for (const key in object) {
+      if (key === 'extraData') {
```

Comment:
You can apply `switch` statement by `action.type` instead of nested `loop` and `if` statements